AutomatticTracks/build.gradle

```diff
@@ -15,11 +15,13 @@ dependencies {
     implementation "io.sentry:sentry-android-okhttp"
     implementation 'androidx.annotation:annotation:1.1.0'
     implementation 'com.squareup.okhttp3:okhttp:4.9.0'
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4'
```

Comment:
Both cliensts user coroutines. I found this approach more effective and readable than custom listeners/emmiters.